unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEmotesSectionController.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL.Emotes;
+using DCL.EmotesCustomization;
+using DCL.Tasks;
+using System;
+using System.Linq;
+using System.Threading;
+using UnityEngine;
+
+namespace DCL.Backpack
+{
+    public class BackpackEmotesSectionController : IBackpackEmotesSectionController
+    {
+        public event Action<string> OnNewEmoteAdded;
+        public event Action<string> OnEmotePreviewed;
+        public event Action<string> OnEmoteEquipped;
+        public event Action<string> OnEmoteUnequipped;
+
+        private DataStore dataStore;
+        private IUserProfileBridge userProfileBridge;
+        private IEmotesCatalogService emotesCatalogService;
+        private IEmotesCustomizationComponentController emotesCustomizationComponentController;
+        private CancellationTokenSource loadEmotesCts = new ();
+
+        public void Initialize(
+            DataStore dataStore,
+            Transform emotesSectionTransform,
+            IUserProfileBridge userProfileBridge,
+            IEmotesCatalogService emotesCatalogService)
+        {
+            this.dataStore = dataStore;
+            this.userProfileBridge = userProfileBridge;
+            this.emotesCatalogService = emotesCatalogService;
+
+            emotesCustomizationComponentController = new EmotesCustomizationComponentController();
+            IEmotesCustomizationComponentView emotesSectionView = emotesCustomizationComponentController.Initialize(
```

Comment:
same initialization, can we inject dependencies?

Reply:
Fixed here: https://github.com/decentraland/unity-renderer/pull/4933/commits/f176b5442e99e5b1b910728c23f6998e0b19d520